worlds/sc2/pool_filter.py

```diff
@@ -267,7 +263,6 @@ def attempt_removal(item: Item) -> bool:
             # No orbital Command Spells
             inventory = [item for item in inventory if item.name != item_names.PLANETARY_FORTRESS_ORBITAL_MODULE]
             unused_items = [item_name for item_name in unused_items if item_name !=item_names.PLANETARY_FORTRESS_ORBITAL_MODULE]
-            locked_items = [item for item in locked_items if item.name != item_names.PLANETARY_FORTRESS_ORBITAL_MODULE]
```

Comment:
Fails tests as items can be locked by min/max amount of upgrades. You need to not remove the item from locked ones only if it's locked by the mission order